api/middleware.go

```diff
@@ -82,10 +82,13 @@ func MetricsMiddleware(m metrics.RequestMetrics, logger log.Logger) func(next ht
 				// like we're doing here if we don't want to use it.
 				timer.ObserveDuration()
 			}
+			latency := time.Since(t)
 			logger.Info("ending request",
 				"endpoint", r.URL.Path,
+				"query_params", r.URL.RawQuery,
```

Comment:
Kinda silly, but: Since these two fields have such different names, I fear that a casual / less informed reader of these logs (including us in a few months :)) might miss one or the other. Can we go with names that share a common prefix? Or just replace both of these with a single field that logs `r.URL.RequestURI()` ([link](https://pkg.go.dev/net/url#URL.RequestURI)). If we ever want to filter by just the path (unlikely, IMO, because the params are so important), there's always regex search.